api/src/main/java/org/openmrs/validator/VisitValidator.java

```diff
@@ -98,6 +102,10 @@ public void validate(Object target, Errors errors) {
 		
 		// check attributes
 		super.validateAttributes(visit, errors, Context.getVisitService().getAllVisitAttributeTypes());
+
+		validateStartDateTimeToBirthdate(visit, errors);
+
+		validateStartDateTimeToEstimatedBirthdate(visit, errors);
```

Comment:
Should I remove "Patient patient = visit.getPatient();" from my methods and add it to VisitValidator.validate? I would be able to check if birthdate is estimated or not and call suitable function.